js/models/Order.js

```diff
@@ -0,0 +1,51 @@
+import BaseModel from './BaseModel';
+import _ from 'underscore';
+import app from '../app';
+
+export default class extends BaseModel {
+  defaults() {
+    return {
+      shipTo: '',
+      address: '',
+      city: '',
+      state: '',
+      postalCode: '',
+      countryCode: '',
+      addressNotes: '',
+      moderator: '',
+      items: [{
```

Comment:
I'll put in a nested collection. It's a little weird now, the protocol allows multiple items but there's no way to actually do that in the UI design.